tests/h/services/bulk_executor/conftest.py

```diff
@@ -31,3 +34,12 @@ def group_upsert_command(n=0, authority=AUTHORITY, query_authority=AUTHORITY, **
     command.body.attributes["authority"] = authority
 
     return command
+
+
+@pytest.fixture
+def user(db_session):
+    user = User(_username="username", authority="lms.hypothes.is")
+    db_session.add(user)
+    db_session.flush()
```

Comment:
The tests fail without this `flush()`, which is where I probably copy-pasta'd it from before.